scripts/test-infra/push-image.sh

```diff
@@ -5,5 +5,8 @@
 # container image tag
 VERSION_OVERRIDE=${_GIT_TAG+VERSION=${_GIT_TAG:10}}
 
-make image $VERSION_OVERRIDE
-make push $VERSION_OVERRIDE
+make image-all $VERSION_OVERRIDE
+make push-all $VERSION_OVERRIDE
```

Comment:
`push-all` always also builds images, right(?) So `make image-all` is kinda redundant here?

```suggestion
make push-all $VERSION_OVERRIDE
```